### DISCUSSION TITLE
Why does `NativeMenuItemSeparator` inherit from `NativeMenuItem` instead of `NativeMenuItemBase`?

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#15923] [Q&A]
Same as title

Avalonia Versions: 2dfd9be66afcbd1a571c707acf4e41e2a857e02e , `11.0.x`, `11.1.0-beta2`

**Comment by maxkatz6** [comment]
Old artifacts, I suppose.

**Reply by maxkatz6** [reply]
Kinda - https://github.com/AvaloniaUI/Avalonia/pull/9655/files. NativeMenuBar was hardcoded to handle "-" as a separator, which was the only way to get native menus working before. 
I don't think this hack is required anymore, so that PR can be reverted. 

Unfortunately, it's a breaking change now.
